rc/filetype/elvish.kak

```diff
@@ -0,0 +1,89 @@
+# Elvish
+# https://elv.sh
+
+hook global BufCreate '.*\.elv' %{
+  set-option buffer filetype elvish
+}
+
+hook global WinSetOption filetype=elvish %{
+  require-module elvish
+  evaluate-commands set-option window static_words %opt{elvish_keywords}
+  add-highlighter window/elvish ref elvish
+  hook -group elvish window InsertChar '\n' elvish-new-line-inserted
+  set-option window extra_word_chars '$' '~' '-'
+  hook -always -once window WinSetOption filetype=.* %{
+    remove-highlighter window/elvish
+    remove-hooks window elvish
+  }
+}
+
+provide-module elvish %{
+  declare-option -hidden str-list elvish_keywords 'all' 'assoc' 'base' 'bool' 'break' 'cd' 'chr' 'constantly' 'continue' 'count' 'dir-history' 'dissoc' 'drop' 'each' 'eawk' 'echo' 'eq' 'esleep' 'eval-symlinks' 'exec' 'exit' 'explode' 'external' 'fail' 'fclose' 'fg' 'float64' 'fopen' 'from-json' 'from-lines' '-gc' 'get-env' 'has-env' 'has-external' 'has-key' 'has-prefix' 'has-suffix' 'has-value' '-ifaddrs' 'is' '-is-dir' 'joins' 'keys' 'kind-of' '-log' 'make-map' 'multi-error' 'nop' 'not' 'not-eq' 'ns' 'one' 'only-bytes' 'only-values' 'ord' 'order' '-override-wcwidth' 'path-abs' 'path-base' 'path-clean' 'path-dir' 'path-ext' 'peach' 'pipe' 'pprint' 'prclose' 'print' 'put' 'pwclose' 'rand' 'randint' 'range' 'read-line' 'read-upto' 'repeat' 'replaces' 'repr' 'resolve' 'return' 'run-parallel' 'search-external' 'set-env' 'slurp' '-source' 'splits' 'src' '-stack' 'styled' 'styled-segment' 'take' 'tilde-abbr' '-time' 'time' 'to-json' 'to-lines' 'to-string' 'unset-env' 'wcswidth'
```

Comment:
I dont like introducing Kakoune options that are only used once, why cant that be directly in the shell block as a shell variable ?